src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -527,16 +556,22 @@ object TezosDatabaseOperations extends LazyLogging {
       predicates: List[Predicate],
       ordering: List[QueryOrdering],
       aggregation: List[Aggregation],
+      temporalPartition: Option[String],
```

Comment:
as described above, adding `temporal-partition` to the metadata configuration marks the table as temporal and then `selectWithPredicates` uses `makeTemporalQuery()` instead of `makeQuery()`